### ISSUE TITLE
Sometimes replies will be "Error, the reference "SessionName" does not exist"

### ISSUE THREAD

**Issue by SquirrelsMcGee** [CleverbotIO/node-cleverbot.io#7]
This happens regardless of what I put into bot.setNick();

**Comment by dtesler** [comment]
Looking into it right now.

**Comment by SquirrelsMcGee** [comment]
Ok thank you, I would like to mention that just like Issue #6, this does not occur all the time, and appears to be random, however the solution of doing `setNick("SessionName");` does not seem to work

**Comment by dtesler** [comment]
It seems to be working for me -- what code are you running?

**Comment by SquirrelsMcGee** [comment]
I am using this code for the cleverbot command for a Discord bot,
```
} else if (command === ";cb") {
		var msg = message.content.substring(4);
		if (para != null) {
			if (cleverbotrunning == false) {
				if (para == "#start") {
					botclever = new cleverbot("dxEY8oGdh8l6M91z", "aTqa71k9duZMiBf5Ur088Tb6TB0vQvMP");
					botclever.setNick("SessionName");
					mybot.reply(message, "Starting CleverBot module");
					cleverbotrunning = true;
				} else {
					mybot.reply(message, "CleverBot is not running, use `;cb #start` to start and `;cb #stop` to stop, use `;cb #help` to see this message again");
				}
			} else {
				if (para == "#stop") {
					mybot.reply(message, "Stopping CleverBot module");
					cleverbotrunning = false;
				} else {
					botclever.ask(msg, function (err, response) {
						mybot.reply(message, response);
					});
				}
			}
		}
	}```

**Comment by dtesler** [comment]
I will close this issue for now, but if anything pops back up feel free to reopen it.  Most fixes are postponed until the 2.0 API release.

-- Also, are you using bot.create in your code?  -- Let's continue this conversation on the [Cleverbot.io Slack](https://slack.cleverbot.io).
